File: Signiture-Backend/kotlin/study/src/main/java/variable/Exam.java
package variable;

public class Exam {

    public Exam() {
        String name = "홍길동";

        int age = 10;       // 기본값: 0
        Integer _age = 20;  // 기본값: null

        System.out.println(name + "(" + age + ")");
        String format = "사용자 이름은: %s";
        String result = String.format(format, name);
        System.out.println(result);
    }

    public static void main(String[] args) {
        new Exam();
    }
}

File: Signiture-Backend/kotlin/study/src/main/kotlin/variable/Exam.kt
package pcy.study.variable

fun main() {
    // var: mutable(가변)
    // val(final): immutable(불변)

    // : [타입]
    val name: String = "홍길동"

    // Kotlin은 모두 Reference 타입
    val age: Int = 10

    println(name + "(" + age + ")")
    val result = "사용자 이름은: $name"
    println(result)
}